### ISSUE TITLE
VST3: GUI not showing up in Studio One

### ISSUE THREAD

**Issue by macay** [bcmodular/scopesync#29]
When I load the VST3 version in Studio One 2.6.3, the GUI is not displayed properly. Actually it's not displayed at all. It seems that it's displayed right after loading, but vanishes immediately afterwards.

![scopesync-vst3-64bit](https://cloud.githubusercontent.com/assets/8532304/4023327/0e1e14ea-2b87-11e4-9b54-143eea7f3c33.png)

This happens with both the 32-bit and 64-bit versions on Windows 7 64-bit and Windows XP.

**Comment by wellis** [comment]
I'm afraid this one will have to be parked for now, as I'm waiting on Juce support for dynamic parameters in VST3 plugins. See: https://forum.juce.com/t/updatehostdisplay-vst3/13338
